openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -113,9 +114,20 @@ def is_enqueue_enabled(cls, site):
         return False
 
     def run(
-        self, site_id, target_day_str, day_offset, bin_num, override_recipient_email=None,
+        self, site_id, target_day_str, day_offset, bin_num=None, override_recipient_email=None,
     ):
         site = Site.objects.select_related('configuration').get(id=site_id)
+
+        # If no target date was provided, then enqueue tasks from the current date
+        if target_day_str is None:
+            return self.enqueue(
+                site,
+                now(),
+                day_offset,
```

Comment:
TODO: to make this more configurable from the `ADDITIONAL_CELERYBEAT_SCHEDULES` setting, could the `day_offset` be a JSON-encoded list of offsets instead of just a single number?  This would let us use the [list of offsets used by the management command](https://github.com/edx/edx-platform/blob/b9bc10e7f232c7d22130123a755573901ee3ba49/openedx/core/djangoapps/schedules/management/commands/send_course_update.py#L8) instead of having to copy-paste a separate entry in `ADDITIONAL_CELERYBEAT_SCHEDULES` for each offset.